### PR TITLE
[NAE-1948] Bugs after merge to 6.4.0

### PR DESCRIPTION
**Pull request by machacjozef**
# Description

Fixes [NAE-1948]

## Dependencies

No new dependencies were introduced

### Third party dependencies

No new dependencies were introduced

### Blocking Pull requests

There are no dependencies on other PR

## How Has Been This Tested?

This was tested manually.

### Test Configuration

| Name                | Tested on |
|---------------------| --------- |
| OS                  |   macOS Ventura 14.0        |
| Runtime             |  Node 14.19.3         |
| Dependency Manager  |  NPM 6.14.17        |
| Framework version   |  Angular 13.3.1         |
| Run parameters      |           |
| Other configuration |           |

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My changes have been checked, personally or remotely, with @machacjozef 
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have resolved all conflicts with the target branch of the PR
- [x] I have updated and synced my code with the target branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes:
    - [ ] Lint test
    - [ ] Unit tests
    - [ ] Integration tests
- [ ] I have checked my contribution with code analysis tools:
    - [ ] [SonarCloud](https://sonarcloud.io/project/overview?id=netgrif_components)
    - [ ] [Snyk](https://app.snyk.io/org/netgrif)
- [x] I have made corresponding changes to the documentation:
    - [x] Developer documentation
    - [x] User Guides
    - [x] Migration Guides

[NAE-1948]: https://netgrif.atlassian.net/browse/NAE-1948?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
     <!-- Small layout -->
     <ng-template #elseTemplate>
-        <div fxLayout="row" fxLayoutAlign="start center" class="fill-all-space" [ngClass]="{'padding-8px': isNotButton()}">
+        <div fxLayout="row" fxLayoutAlign="start center" class="fill-all-space" [ngClass]="{'padding-8px': !isButton(), 'padding-16px': isButton()}">
```
Comment (Kovy95) [inline projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html:20]: why we putting the 16px paddign there ? is the alignment of buttons with fields still ok ?

**Review by Kovy95 — CHANGES_REQUESTED** [review]

File: projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html (hunk 1 above)
Comment (Kovy95) [reply projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html:20]: @machacjozef @renczesstefan

**Review by minop — DISMISSED** [review]

File: projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html (hunk 1 above)
Comment (tuplle) [reply projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html:20]: Can you please attach some screenshots for clearance?  🙏  Thank you

**Review by tuplle — DISMISSED** [review]

File: projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html (hunk 1 above)
Comment (Kovy95) [reply projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html:20]: @renczesstefan but you need to chech the alignment of buttons (if they are aligned with text field for example), there is a reason that the padding was removed

Comment (renczesstefan) [reply projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html:20]: This is how it looks like without this PR 
image

this is how it looks like with original padding but with no shadows
image

and this is how it looks like with new padding and with no shadow
image

We should decide which one to chose.

Comment (machacjozef) [reply projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html:20]: @tuplle & @Kovy95

Comment (Kovy95) [reply projects/netgrif-components/src/lib/data-fields/data-field-template/data-field-template.component.html:20]: padding 8px from left and right and any padding from top and bottom

**Review by Kovy95 — DISMISSED** [review]

**Review by Kovy95 — APPROVED** [review]

**Pull request merged**